.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "mobilecoin"]
 	path = mobilecoin
 	url = https://github.com/mobilecoinfoundation/mobilecoin.git
-	
+
 [submodule "ledger-mob"]
 	path = ledger-mob
-	url = https://github.com/mobilecoinofficial/ledger-mob.git
+	url = git@github.com:eranrund/ledger-mob.git
```

Comment:
This needs to be reverted, I have some small changes to the `ledger-mob` repository but they are currently gated by https://github.com/mobilecoinfoundation/mobilecoin/pull/4030 getting merged, so I am punting on creating the `ledger-mob` PR.